src/libraries/sendtohelix.proj

```diff
@@ -40,6 +40,12 @@
     <DotNetCliVersion>$([System.Text.RegularExpressions.Regex]::Match($(GlobalJsonContent), '(%3F&lt;="dotnet": ").*(%3F=")'))</DotNetCliVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetsMobile)' == 'true'">
+    <ArchiveTests>false</ArchiveTests>
```

Comment:
This ArchiveTests doesn't do anything I think. It was my attempt that didn't work and then you resolved it in yaml